data/json/snippets.json

```diff
@@ -89,6 +94,16 @@
     "category": "flier",
     "text": [ "This is an advertisement for Sybil's Cyber Boutique.  It shows a picture of a man flexing shiny cybernetic arms in front of a mirror.  He is flanked by two scantily clad nurses who hold bloody chainsaws and give approving thumbs up.  In the background feral dogs gnaw on the man's discarded former limbs.  The caption reads:  \"Spring Cleaning Sale!  20% off!\"" ]
   },
+  {
+    "type": "snippet",
+    "category": "flier",
+    "text": [ "This is an advertisement for Sybil's Cyber Boutique.  It shows a picture of a man carrying a somewhat confused horse on his shoulders.  He is flanked by two scantily clad nurses who hold bloody chainsaws and give approving thumbs up.  His bulging knees and elbows are stiched and seem to be the source of his ability.  The caption reads:  \"Brand new!  Horse power at hand!\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "flier",
+    "text": [ "This is an advertisement for Sybil's Cyber Boutique.  It shows a picture of a man smiling while being relentlesly punched by a boxer.  He doesn't even sweat as punches have no effect on him.  Two scantily clad cheerleader nurses give approving thumbs up from the background.  The caption reads:  \"Bouncy as never before!  Absorb those shocks!\"" ]
```

Comment:
What is a cheerleader nurse?

Reply:
I'd post a picture but that would get me kicked. But ok, lest cut out the confusion and just leave nurses.